imqsauth/http.go

```diff
@@ -81,12 +83,15 @@ type userGroupsResponseJson struct {
 }
 
 type ImqsCentral struct {
-	Config    *Config
-	Central   *authaus.Central
-	Yellowfin *Yellowfin
+	Config               *Config
+	Central              *authaus.Central
+	Yellowfin            *Yellowfin
+	WebSocketConnections map[int]*websocket.Conn
```

Comment:
There should be a comment explaining what this map stores, and what the integer key means. For a good example, see the TLSNextProto comment in https://golang.org/pkg/net/http/#Server. Also, a better name might be ChangeListeners, because we're more interested in what they're doing, than the transport mechanism that they use.

The 'id' variable here needs a better name. We need to know that it is associated with websocket listeners.

Both WebSocketConnections and 'id' need to be guarded by a synchronization mechanism. Remember that all server code you write here is multithreaded. I would probably just use a plain old mutex.